### ISSUE TITLE
On “High Contrast On” mode, the name "Untitled" is not clear

### ISSUE THREAD

**Issue by Helen09** [microsoft/pxt-ev3#477]
**Repro Steps:**
• Navigate to https://makecode.mindstorms.com/beta#editor
• Click "home" to back to Home Page
• Click "More" -> "High Contrast On" and then do the second step again

**Expected result:**
On "High Contrast On" mode, the name "Untitled" should be clear
**Actual result:**
On “High Contrast On” mode, the name "Untitled" is not clear
![7](https://user-images.githubusercontent.com/23466737/38540432-7f7dc4dc-3c8b-11e8-9a78-8422513efcfd.gif)
